examples/snippet_coloredcout_main.cpp

```diff
@@ -18,12 +18,14 @@ int main() {
       LOG(INFO) << "An INFO message in white";
       LOG(WARNING) << "A WARNING message will be in yellow";
 #if 0
-      LOG(FATAL) << "This call will exit the example. A failed CHECK call, \n"
-                 << "a LOG(FATAL) or a fatal crash (seg fault or similar) will show up in RED\n"
-                 << "\n\nThis call will also be DUPLICATED. One FATAL LOG message will be shown through the \n"
-                 << "the crash handler and sent to cout, the LOG call will asynchronously send the LOG output to cout.\n"
```

Comment:
Oh ops the formatting wasn't supposed to be pushed, the code is that it has a missing ";" so that's why the line was modified. I can keep it that way if you want.

Reply:
thanks for the observation and fix